composer.json

```diff
@@ -213,7 +213,7 @@
         "drupal/key": "^1.10",
         "drupal/key_auth": "^2.1",
         "drupal/key_value_field": "^1",
-        "drupal/linkit": "^6",
+        "drupal/linkit": "^6.0",
```

Comment:
See Jira ticket. I dont think this PR will be needed once we put back the Pathologic patch we have always used.